app/src/main/java/com/nextcloud/talk/controllers/ConversationInfoController.kt

```diff
@@ -529,8 +558,10 @@ class ConversationInfoController(args: Bundle) :
 
                         if (conversationCopy!!.canModerate(conversationUser)) {
                             binding.addParticipantsAction.visibility = View.VISIBLE
+                            binding.clearConversationHistory.visibility = View.VISIBLE
```

Comment:
Needs to be wrapped by a capability check for `clear-history`

Ref https://github.com/nextcloud/spreed/pull/6023